module-system/module-implementations/sov-nft-module/src/call.rs

```diff
@@ -87,6 +88,8 @@ impl<C: Context> NonFungibleToken<C> {
         )?;
         self.collections
             .set(&collection_address, &collection, working_set);
+        std::thread::sleep(std::time::Duration::from_millis(300));
```

Comment:
Two comments here: 
1. If we're going to introduce a sleep into code, we should definitely add a comment explaining it! 
2. I don't think we can safely add a sleep into the critical path of processing a call message. I don't want to block this tutorial on something unrelated to its functionality (adding synchronization to the sequencer), but I don't feel good about this. Users running the NFT demo will see 3 tps, and it won't be obvious why

Reply:
oh yeah, this was just to illustrate the issue but it got checked in accidentally. removed now!